src/messaging/tests.py

```diff
@@ -114,6 +114,19 @@ def get_thread(self):
         self.assertEqual(self.message1_json, resp_message2)
         self.assertEqual(self.message2_json, resp_message2)
 
+    def get_thread_user_DNE(self):
+        bad_id = random.randInt(3,100)
+        while bad_id == self.me.id | bad_id == self.other.id:
+            bad_id = random.randInt(3,100)
```

Comment:
why not just `bad_id = self.me.id + self.other.id`?

also, `|` isn't good practice in python.  I think it might actually be bitwise.  `or` should always be used for boolean `OR`